qbraid_qir/qiskit/visitor.py

```
@@ -0,0 +1,331 @@
+    def visit_register(self, register: Union[QuantumRegister, ClassicalRegister]) -> None:
+        """Visit a register and assign labels to its bits.
+
+        Args:
+            register: The quantum or classical register to visit.
+        """
+        logger.debug("Visiting register '%s'", register.name)
+        if isinstance(register, QuantumRegister):
+            self._qubit_labels.update(
+                {bit: n + len(self._qubit_labels) for n, bit in enumerate(register)}
+            )
+            logger.debug("Added labels for qubits %s", [bit for bit in register])
```

Ruff (0.15.2)

[convention] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)